tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot

```diff
@@ -14,7 +14,7 @@ ${commercial_url}              https://www.redhat.com/en/technologies/cloud-comp
 *** Test Cases ***
 Verify RHODS operator information
   [Documentation]  This TC verfiy if the text present in RHODS opeartor Details section
-  [Tags]  ODS-498   ODS-624   Sanity
+  [Tags]  ODS-498   ODS-624   Sanity    FlakyTest
```

Comment:
i would suggest to have tags as defined in guidline.
example:
[Tags]  Sanity
    ...     ODS-498
    ...     ODS-624
    ...     FlakyTest

Reply:
As the PR already has one approval, do you mind if I address it in a future PR?

Reply:
no worries. I will approve as well